.gitignore

```diff
@@ -2,6 +2,7 @@
 log/*.log
 pkg/
 node_modules/
+coverage/
```

Comment:
Hey, we can't accept this. Please add it to its own PR and explain why you'd want to do it - right now I think we keep the `coverage/` to show it off in a badge sometime soon :-)